crates/ruff_workspace/src/resolver.rs

```diff
@@ -236,39 +209,31 @@ fn resolve_configuration(
     while let Some(extend) = stack.pop() {
         configuration = configuration.combine(extend);
     }
-    processor.process_config(&mut configuration);
```

Comment:
I think I consider adding `base_path` to `Configuration`, and always overwrite it in `combine`. It seems important to know with which `base_path` the configuration was resolved and it must be the same as we pass to `into_all_settings`

@charliermarsh any thoughts on this?